android/src/main/java/io/customer/reactnative/sdk/messaginginapp/BaseInlineInAppMessageViewManager.kt

```diff
@@ -0,0 +1,45 @@
+package io.customer.reactnative.sdk.messaginginapp
+
+import com.facebook.react.uimanager.SimpleViewManager
+import com.facebook.react.uimanager.ThemedReactContext
+import com.facebook.react.uimanager.annotations.ReactProp
+
+/**
+ * Base view manager for inline in-app message components.
+ *
+ * Provides common functionality for both old and new React Native architecture
+ * implementations, including view creation, event handling, and property management.
+ */
+abstract class BaseInlineInAppMessageViewManager :
+    SimpleViewManager<ReactInlineInAppMessageView>() {
+    override fun getName() = NAME
+
+    override fun createViewInstance(context: ThemedReactContext): ReactInlineInAppMessageView {
```

Comment:
do we have a callback for releasing resources?

Reply:
I didn't see one in React Native guides, but I'll take another look at `ViewManager` docs and implement it if needed.